outputs.tf

```diff
@@ -28,6 +27,20 @@ output "git_clone_url" {
   description = "URL that can be provided to git clone to clone the repository anonymously via the git protocol."
 }
 
+# ----------------------------------------------------------------------------------------------------------------------
+# OUTPUT ALL RESOURCES AS FULL OBJECTS
+# ----------------------------------------------------------------------------------------------------------------------
+
+output "repository" {
+  value       = github_repository.repository
+  description = "All attributes and arguments as returned by the github_repository resource."
+}
+
+output "branches" {
+  value       = github_branch.branch
+  description = "A map of branch objects keyed by branch name."
+}
+
```

Comment:
I think this needs to be added to the outputs section in the documentation.

Reply:
that's correct, will take care of it

Reply:
added in 32c22f8a